web/client/themes/default/icons.less

```diff
@@ -44,966 +44,651 @@
   -moz-osx-font-smoothing: grayscale;
 }
 
-.glyphicon-1-bearing:before {
```

Comment:
why this big rewrite of icons.less? It0s automatic?

Reply:
Yes, they are generated automatically generated based on this [script](https://github.com/geosolutions-it/MapStore2/blob/master/build/generateIcons.js).
I need to test a different approach, I'll submit a new commit if it works.

Reply:
ok, don't forget to document icon generation procedure with that update.